scripts/local-test.sh

```diff
@@ -35,14 +35,6 @@ function stopCluster() {
 
 readonly SRCDIR=$(readlink -f ${BASH_SOURCE[0]%/*})
 
-# Install dependencies
-pip install -r requirements_dev.txt
```

Comment:
these should be installed by developer manually - it only needs to be done once as part of environment setup.